packages/plugin-e2e/src/models/components/Select.ts

```diff
@@ -0,0 +1,41 @@
+import { Locator } from '@playwright/test';
+import { ComponentBase } from './ComponentBase';
+import { SelectOptionsType } from './types';
+
+export class Select extends ComponentBase {
+  constructor(element: Locator) {
+    super(element);
+  }
+
+  async selectOption(values: string, options?: SelectOptionsType): Promise<string> {
+    const menu = await openSelect(this.element, options);
+    return selectByValueOrLabel(values, menu, options);
+  }
+}
+
+export async function openSelect(element: Locator, options?: SelectOptionsType): Promise<Locator> {
+  await element.getByRole('combobox').click(options);
+  return element.page().getByLabel('Select options menu', {
```

Comment:
Use `selectors.components.Select.menu` instead

Reply:
A slightly different selector got added in 11.4

https://github.com/grafana/grafana/blob/main/packages/grafana-e2e-selectors/src/selectors/components.ts#L881

Reply:
Let's use the selector here too